educe/stac/learning/features.py

```diff
@@ -1411,12 +1372,12 @@ def _fuse_corpus(corpus, postags):
     to_delete = []
     for key in corpus:
         if key.stage == 'unannotated':
-            corpus[key] = _fuse_edus(corpus[key], corpus[key], postags[key])
+            corpus[key] = fuse_edus(corpus[key], corpus[key], postags[key])
```

Comment:
This kind of call, with the same document passed as discourse and unit annotations, abuses fuse_edus.
This could indicate that fuse_edus does not have the right perimeter of responsibility and some refactoring could be useful.
As this is not urgent, maybe you could add a comment here, that briefly touches on this abuse?